src/Utilities/PaymentBody.res

```diff
@@ -547,14 +547,20 @@ let applePayThirdPartySdkBody = (~connectors) => [
   ),
 ]
 
-let cryptoBody = (~currency) => [
+let cryptoBody = (~currency, ~cryptoCurrencyNetworks) => [
   ("payment_method", "crypto"->JSON.Encode.string),
   ("payment_method_type", "crypto_currency"->JSON.Encode.string),
   ("payment_experience", "redirect_to_url"->JSON.Encode.string),
   (
     "payment_method_data",
     [
-      ("crypto", [("pay_currency", currency->JSON.Encode.string)]->Utils.getJsonFromArrayOfJson),
+      (
+        "crypto",
+        [
+          ("pay_currency", currency->JSON.Encode.string),
+          ("network", cryptoCurrencyNetworks->JSON.Encode.string),
```

Comment:
This should be handled in the dynamic fields

Reply:
Updated